test/e2e/tests/request-queuing/dapp1-switch-dapp2-send.spec.js

```diff
@@ -80,9 +76,6 @@ describe('Request Queuing Dapp 1, Switch Tx -> Dapp 2 Send Tx', function () {
           css: 'p',
         });
 
-        // Wait for the first dapp's connect confirmation to disappear
-        await driver.waitUntilXWindowHandles(2);
-
```

Comment:
no need to add this, as we handle the close window into the `clickElementAndWaitForWindowToClose